check_http_json.rb

```diff
@@ -18,23 +18,59 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-
-
 # Requires.
-require 'rubygems'
 require 'json'
 require 'net/http'
 require 'net/https'
 require 'uri'
 require 'optparse'
 require 'timeout'
 
+# Manage Nagios messages and exit code
+module Nagios
```

Comment:
Because this is a self instance which is required to hold the current state of the code level. 
A module is accessible everywhere in the code, which reduces the complexity of usage ;-)